docs/experimental-features/investigation-guide-actions.asciidoc

```diff
@@ -0,0 +1,80 @@
+[[interactive-investigation-guides]]
+== Interactive investigation guides
+
+NOTE: This feature is available for {stack} versions 8.6.0 and newer.
+
+Detection rule investigation guides suggest steps for triaging, analyzing, and responding to potential security issues. For custom rules, you can create an interactive investigation guide that includes buttons for launching runtime queries in <<timelines-ui,Timeline>>, using alert data and hard-coded literal values. This allows you to launch detailed Timeline investigations directly from an alert using relevant data. 
+
+[role="screenshot"]
+image::images/ig-alert-flyout.png[Alert details flyout with interactive investigation guide,550]
+
+Each query button displays the number of documents found. Click the button to automatically create the query in Timeline based on the investigation guide's configuration.
+
+[role="screenshot"]
+image::images/ig-timeline.png[Timeline with query pre-loaded from investigation guide action]
+
+[discrete]
+[[add-ig-actions-rule]]
+=== Add investigation guide actions to a rule
+
+NOTE: You can only create interactive investigation guides with custom rules because Elastic prebuilt rules can't be edited. However, you can duplicate a prebuilt rule, then configure the investigation guide for the duplicated rule.
+
+You can configure an interactive investigation guide when you <<rules-ui-create,create a new rule>> or <<edit-rules-settings,edit an existing rule>>. When configuring the rule's settings (the *About rule* step for a new rule, or the *About* tab for an existing rule), expand the *Advanced settings*, then scroll down to the *Investigation guide* field.
+
+[role="screenshot"]
+image::images/ig-investigation-guide-editor.png[Investigation guide editor field]
+
+Add the following syntax to the *Investigation guide* field to configure a query. The syntax will appear as an action button in the investigation guide.
+
+[width="100%",options="header"]
+|===
+|Field |Description
+
+|`!{insight{ }}` |The container object holding all the configuration attributes for the button and the query.
+|`label` |Label text on the button.
+|`description` |Description text on the button.
```

Comment:
Could this be something less literal, for ex, "Additional information you want to include after the button label"? Or is this a case when it's better to be more literal?

Reply:
Added an [edit](https://github.com/elastic/security-docs/pull/2821/files#r1069633653) to address this and the previous line's comment. I wanted to keep "text" because just calling something a "label" sounds unclear to me.

Reply:
edit lgtm 🚀